plenum/config.py

```diff
@@ -413,3 +413,10 @@
 TXN_AUTHOR_AGREEMENT_ACCEPTANCE_TIME_AFTER_PP_TIME = 120
 
 NEW_VIEW_TIMEOUT = 30  # in secs
+
+# Flags
+TRANSPORT_BATCH_ENABLED = True
```

Comment:
I have a suspicion that this flag needs to be placed in stp_core config...

Reply:
Currently Batching logic is in Plenum, not ZMQ...